Triangulation_3/include/CGAL/Triangulation_hierarchy_3.h

```diff
@@ -578,7 +578,7 @@ insert_and_give_new_cells(const Point &p, OutputItCells fit, Cell_handle start)
   Locate_type lt;
   int i, j;
   // locate using hierarchy
-  locs positions[maxlevel];
+  locs positions[maxlevel] = {-1};
```

Comment:
Note that this initializes the first element to -1, and the others to 0, IIRC.

Reply:
Thank you for allowing this to roll forward.

-1 does not work anyway, as 'locs' is a struct. (line 440).
The alternative here would be to default the values in the struct.

I did a quick test for the failure in my codebase and found that initializing the members li and lj with -1, was sufficient to address the issue.

Reply:
I think `locs positions[maxlevel] = {-1,-1,-1,-1};` would have worked (Struct initialization), but the  solution you proposed is better.